extensions/graphite-emitter/src/main/java/io/druid/emitter/graphite/GraphiteEvent.java

```diff
@@ -0,0 +1,39 @@
+package io.druid.emitter.graphite;
+
+import javax.validation.constraints.NotNull;
+
+public class GraphiteEvent
+{
+  private final String eventPath;
+  private final String value;
+  private final long timestamp;
+
+  /**
+   * A graphite event must be in the following format: <metric path> <metric value> <metric timestamp>
+   *   ex:  PRODUCTION.host.graphite-tutorial.responseTime.p95 0.10 1400509112
+   * @param eventPath This is the namespace path of the metric
+   * @param value value of the metric
+   * @param timestamp unix time in second
+   */
+  GraphiteEvent(@NotNull String eventPath, @NotNull String value, @NotNull long timestamp)
+  {
+    this.eventPath = eventPath;
+    this.value = value;
+    this.timestamp = timestamp;
+  }
```

Comment:
`@NotNull` probably does not do anything, you would have to use Preconditions check and `long timestamp` can never be null